umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/dif10.clj

```diff
@@ -197,6 +198,25 @@
         :OnlineResource (when-let [linkage (value-of data-set-citation "Online_Resource")]
                           {:Linkage linkage})}))))
 
+(defn- add-distribution-infos
+  "This function creates a distribution information map."
+  [media size unit fees format]
+  {:Media media
+   :AverageFileSize size
+   :AverageFileSizeUnit unit
+   :Format format
+   :Fees fees
+   :FormatType "Native"})
+
+(defn- add-dist-info-for-multi-formats
+  "This method takes the value of a format and parses it into multiple formats if the metadata includes multiple formats. Then this function creates new distribution information maps for each format."
+  [media size unit formats fees]
+  (when (or media size unit formats fees)
+    (let [distributions
+          (map #(add-distribution-infos media size unit fees %) (format-util/parse-distribution-formats formats))]
+      (when distributions
```

Comment:
Is this `when` necessary? I feel like this should always be populated

Reply:
No it is not necessary.  I have removed it.